packages/panels/src/Navigation/NavigationItem.php

```diff
@@ -145,9 +145,11 @@ public function sort(int | Closure | null $sort): static
         return $this;
     }
 
-    public function url(string | Closure | null $url, bool | Closure $shouldOpenInNewTab = false): static
+    public function url(string | Closure | null $url, bool | Closure | null $shouldOpenInNewTab = null): static
     {
-        $this->openUrlInNewTab($shouldOpenInNewTab);
+        if ($shouldOpenInNewTab !== null) {
+            $this->openUrlInNewTab($this->shouldOpenUrlInNewTab() || $shouldOpenInNewTab);
```

Comment:
This isn't required, right? We shouldn't call a getter from a setter

```suggestion
            $this->openUrlInNewTab($shouldOpenInNewTab);
```

Reply:
You are right, I only added it there, because $shouldOpenInNewTab can be a closure, and I did not want to copy the logic to evaluate it, but you are right, there is no need to evaluate it right away. I arrived at this after running some circles and did not notice this